liquidity-module/index.md

```diff
@@ -35,22 +35,30 @@ You can follow the [code with us session](https://www.youtube.com/watch?v=Gxaqpz
 
 Before you start the tutorial, install the prerequisite software. 
 
-- [Install Starport](../starport/index.md) v0.16.2
+- [Install Starport](../starport/index.md) v0.16.2 or later
 
-    **Important** This tutorial uses [Starport](https://github.com/tendermint/starport) v0.16.2. The tutorial is based on this specific version of Starport and is not supported for other versions.
+    **Important** This tutorial uses [Starport](https://github.com/tendermint/starport) v0.16.2. The tutorial is based on this specific version of Starport and is not supported for older versions.
 
 - Install the Gravity DEX binary
 
     - Clone the repo:
 
     ```bash
-    git clone https://github.com/b-harvest/gravity-dex.git
+    git clone https://github.com/cosmos/gaia && cd gaia
     ```
     
-    - Move into the directory and install software to the right location:  
+    - Check the current version 
 
     ```bash
-    cd gravity-dex && make install
+    git checkout v5.0.5
+    ```
+    
+    Note: You can find latest version here - https://github.com/cosmos/gaia/releases
```

Comment:
Let's remove this note, as we want them to use this specific version v5.0.5

Reply:
```suggestion
```